data/surface-pro-2017.tablet

```diff
@@ -0,0 +1,19 @@
+# USB tablet PC models:  HP TouchSmart TX2z, Dell Latitude XT, and Dell Latitude XT2
+#
+# stylus with two buttons and no eraser; 4FGT touchscreen
+#
+# HP Screen size 10.2 x 6.4; 12.1" diagonal
+# Dell Screen size 10.3 x 6.4; 12.1" diagonal
+
+[Device]
+Name=Surface Pro 2017
+DeviceMatch=mei:1b96:001F
+Class=ISDV4
+Width=10
+Height=6
+IntegratedIn=Display;System
+
+[Features]
+Stylus=true
+Touch=true
+Buttons=0
```

Comment:
Unless I'm misreading, this file is deleted in the next commit? Can we squash the two together please?

Also, if you haven't done so yet, please submit the HID output to https://github.com/linuxwacom/wacom-hid-descriptors/, makes it easier to debug in the future.